packages/lib/mattermost/service.ts

```
@@ -0,0 +1,81 @@
+
+    if (!response.ok) {
+      throw new Error("Network response was not ok");
+    }
+
+    const data = await response.text();
+
+    if (data !== "ok") {
+      throw new Error(data);
+    }
+  } catch (error) {
+    throw error;
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)